_conferences/womenwhocodemobilesummit.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Women Who Code Mobile Summit 2022"
+website: https://hopin.com/events/women-who-code-mobile-summit/registration
+location: Online
+online: true
+
+date_start: 2022-07-27
+date_end: 2022-07-27
+
+start:  2022-07-27
+end:    2022-07-27
+site:   https://hopin.com/events/women-who-code-mobile-summit/registration
```

Comment:
These lines are part of the `cfp` section and should have been removed also.

```suggestion
start:  2022-07-27
end:    2022-07-27
site:   https://hopin.com/events/women-who-code-mobile-summit/registration
```

Reply:
Makes sense :) I also like the name suggestion. Thanks for reviewing.